<launch>

  <!-- defines name of the namespace of the drone -->
  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav1)" />

    <!-- Namespace - all topics, services and parameters will be remapped using this namespace as a prefix (eg.: "/waypoint_flier_simple/odom_uav_in" to "/uav1/waypoint_flier_simple/odom_uav_in") -->
  <group ns="$(arg UAV_NAME)">

    <node pkg="motion_planning_python" type="sweeping_generator.py" name="sweeping_generator" output="screen">

        <!-- ROS parameters config file -->
      <rosparam file="$(find motion_planning_python)/config/sweeping_generator.yaml" />

      <remap from="~control_manager_diag_in" to="control_manager/diagnostics" />

      <remap from="~path_out" to="trajectory_generation/path" />

      <remap from="~start_in" to="~start" />

    </node>

  </group>

</launch>
